.env.runtime.schema

```
@@ -0,0 +1,33 @@
+
+# GitHub token for feedback feature
+# @sensitive @required=$USE_FEEDBACK
+NUXT_FEEDBACK_GITHUB_TOKEN=protonPass(pass://DCC-KI/FEEDBACK_DEV_GITHUB_TOKEN/TOKEN)
+
+# Use dummy data (default: false)
+# @type=string
+DUMMY=false
+
+# Logger level (default: debug)
+# @optional @public @type=enum(trace, debug, info, warn, error, fatal)
+NUXT_PUBLIC_LOGGER_LOG_LEVEL=if(eq($APP_MODE, dev), debug, info)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)